doc/release.md

```diff
@@ -27,21 +23,41 @@ git commit -am "Upgrade the version to x.y.z+1-dev0 following x.y.z release"
 
 Check the result and then upgrade the main branch
```

Comment:
```suggestion
7. Move the maintenance branch up to a dev version with `tbump`:

``bash
 tbump X.Y.Z+1-dev0 --no-tag --no-push # You can interrupt during copyrite
 git commit -am "Upgrade the version to x.y.z+1-dev0 following x.y.z release"
``

For example:
``bash
 tbump 2.4.1-dev0 --no-tag --no-push # You can interrupt during copyrite
 git commit -am "Upgrade the version to 2.4.1-dev0 following 2.4.0 release"
``
```

Reply:
Using "``" for code block because otherwise the suggestion breaks. This should be fixed manually.